docker.go

```diff
@@ -284,7 +284,17 @@ func (c *DockerContainer) ContainerIP(ctx context.Context) (string, error) {
 		return "", err
 	}
 
-	return inspect.NetworkSettings.IPAddress, nil
+	ip := inspect.NetworkSettings.IPAddress
+	if ip == "" {
+		for _, settings := range inspect.NetworkSettings.Networks {
+			if settings.IPAddress != "" {
+				ip = settings.IPAddress
```

Comment:
What do you think about using a more descriptive variable name?
```suggestion
		for _, network := range inspect.NetworkSettings.Networks {
			if network.IPAddress != "" {
				ip = network.IPAddress
```

Reply:
Good point!